fairyfishgui.py

```diff
@@ -213,14 +214,15 @@ def __init__(self):
         sg.ChangeLookAndFeel('GreenTan')
 
         board_controls = [[sg.Button('New Game', key='_newgame_'), sg.Button('Load variants', key='_variants_')],
-                        [sg.Button('Load engine', key='_engine_'), sg.Button('Engine on/off', key='_toggle_')],
-                        [sg.Button('Set FEN', key='_set_fen_'), sg.Button('Reset', key='_reset_')],
-                        [sg.Button('Move', key='_move_'), sg.Button('Undo', key='_undo_')],
-                        [sg.Text('Move List')],
-                        [sg.Multiline(do_not_clear=True, autoscroll=True, size=(25, 10), key='_movelist_')],
-                        [sg.Text('Engine Output')],
-                        [sg.Multiline(do_not_clear=True, autoscroll=True, size=(25, 10), key='_engine_output_')],
-                        ]
+                          [sg.Button('Load engine', key='_engine_'), sg.Button('Engine on/off', key='_toggle_')],
+                          [sg.Button('Load NNUE', key='_nnue_'), sg.Button('Set Threads',key='_thread_')],
```

Comment:
Long-term I would suggest to make an engine settings dialogue accessible via the menu bar and/or a button. However, for now I would also be fine with this solution, so I leave it up to you if you want to try to change it.